config/samples/mariadb/mariadb-db.yaml

```
@@ -0,0 +1,207 @@
+    my.cnf: |
+      [mysqld]
+      bind-address = 0.0.0.0
+      default_storage_engine = InnoDB
+      binlog_format = row
+      innodb_autoinc_lock_mode = 2
+      innodb_buffer_pool_size = 256M
+      max_allowed_packet = 256M
+      max_connections = 100
+      character-set-server = utf8mb4
+      collation-server = utf8mb4_unicode_ci
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)